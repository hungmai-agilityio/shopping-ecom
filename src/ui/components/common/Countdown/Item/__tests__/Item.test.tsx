import CountdownItem from '@/ui/components/common/Countdown/Item';
import { render, screen } from '@testing-library/react';

describe('CountdownItem component', () => {
  test('Should render CountdownItem', () => {
    render(<CountdownItem label="Hours" value="7" />);

    const label = screen.getByText('Hours');

    expect(label).toBeInTheDocument();
  });

  test('should be render match to snapshot', () => {
    const { container } = render(<CountdownItem label="Hours" value="7" />);
    expect(container).toMatchSnapshot();
  });
});
