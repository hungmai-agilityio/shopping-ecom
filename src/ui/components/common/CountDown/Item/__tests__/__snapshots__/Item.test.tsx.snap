// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountdownItem component should be render match to snapshot 1`] = `
<div>
  <div
    class="flex flex-col items-center"
  >
    <span
      class="lg:text-lg text-xs font-medium"
    >
      Hours
    </span>
    <span
      class="lg:text-3xl text-xl font-bold"
    >
      7
    </span>
  </div>
</div>
`;
